digits/model/tasks/caffe_train.py

```diff
@@ -279,6 +279,7 @@ def save_prototxt_files(self):
         ### Write solver file
 
         solver = caffe_pb2.SolverParameter()
+        solver.solver_type = getattr(solver, self.solver_type)
```

Comment:
Oh I see what you did. That's clever. Maybe add a comment explaining that you're getting the enum value?